src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -362,7 +363,7 @@ internal static bool HandleClearButtonTouched(this EditText? platformView, Touch
 			if (buttonWidth <= 0)
 				return false;
 
-			var x = motionEvent.GetX();
+			var x = motionEvent.RawX;
```

Comment:
This is the main change.

Reply:
Gonna need an explanation of how this doesn't regress to the problem from #9954, then.

Reply:
That commit was made for a reason; `RawX` is the wrong property to use here. Changing it back will re-break #9954. 

Here's a change that will fix both: https://github.com/dotnet/maui/tree/fix-17453-2
Grab those changes and turn the sandbox page into a UI test.